sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosVectorEmbedding.java

```diff
@@ -84,7 +84,7 @@ public CosmosVectorEmbedding setDataType(CosmosVectorDataType dataType) {
      *
      * @return dimensions
      */
-    public Long getDimensions() {
+    public Integer getDimensions() {
```

Comment:
API review will flag it as breaking right? 
In general, how strict are they on nittigrities?

Reply:
Talked with the Srikanta, and added a new api called `get/setEmbeddingDimensions(Integer)` and deprecated the older api with long. Will remove the older api's in 2-3 months in a minor version.